app/src/main/java/org/jellyfin/androidtv/search/TextSearchFragment.java

```diff
@@ -0,0 +1,73 @@
+package org.jellyfin.androidtv.search;
+
+import android.os.Bundle;
+import android.text.Editable;
+import android.text.TextWatcher;
+import android.view.KeyEvent;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.inputmethod.EditorInfo;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import org.jellyfin.androidtv.R;
+
+import androidx.fragment.app.Fragment;
+import androidx.leanback.app.RowsSupportFragment;
+
+public class TextSearchFragment extends Fragment implements TextWatcher, TextView.OnEditorActionListener {
+    private SearchProvider searchProvider;
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        searchProvider = new SearchProvider(getContext(), getActivity().getIntent().getBooleanExtra("MusicOnly", false));
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.fragment_search_text, container, false);
+    }
+
+    @Override
+    public void onActivityCreated(Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+
+        // Add event listeners
+        EditText searchBar = getActivity().findViewById(R.id.search_bar);
+        searchBar.addTextChangedListener(this);
+        searchBar.setOnEditorActionListener(this);
+
+        // Set up result fragment
+        RowsSupportFragment rowsSupportFragment = (RowsSupportFragment) getChildFragmentManager().findFragmentById(R.id.results_frame);
+        rowsSupportFragment.setAdapter(searchProvider.getResultsAdapter());
```

Comment:
The click listener needs added here to be able to select search results.

Reply:
No idea how I forgot this, probably made a mistake when copy pasting it. It is fixed now :)